src/dsmlp/ext/kube.py

```diff
@@ -16,9 +18,38 @@ def get_namespace(self, name: str) -> Namespace:
         api = self.get_policy_api()
         v1namespace: V1Namespace = api.read_namespace(name=name)
         metadata: V1ObjectMeta = v1namespace.metadata
+        
+        gpu_quota = 1
+        if metadata.annotations is not None and GPU_LIMIT_ANNOTATION in metadata.annotations:
```

Comment:
what happens if metadata is none (we're not 100% sure of the API)? I'd add that case in and write a test

Reply:
Hanzen will add this test case

Reply:
Test case added. Just need to resolve low priority gpu bug before PR can be made.

Reply:
Low priority bug fixed